src/api-client.ts

```diff
@@ -59,22 +59,39 @@ class ApiClient {
     return res.data;
   }
 
-  public async getDeploymentSteps(deploymentLogId: string) {
+  public async getDeployment(
+    deploymentLogId: string,
+    abortController?: AbortController
+  ) {
+    const res = await this.instance.get(
```

Comment:
no need to shorten `response` to `res`. please be consistent across a file

Reply:
24dbf1ce4ada2bd10c97bad950028b9cb31553d2